src/components/new-request/name-check/name-check.vue

```diff
@@ -634,7 +661,7 @@ export default class NameCheck extends Vue {
       return { color: 'red darken-2', icon: 'mdi-alert-octagon' }
     }
     if (this.hasIssuesConflictCaution || this.hasIssuesStructureCaution) {
-      return { color: 'orange darken-2', icon: 'mdi-alert' }
+      return { color: '#F8661A', icon: 'mdi-alert' }
```

Comment:
looks like there is an app colour defined for this already called `$caution`

Reply:
how do I access that variable here?

Reply:
nvm severin mentioned changing the vuetify 'warning' in override -- will do that

Reply:
Add it to Vuetify theme, or replace warning colour. See vuetify.ts.

Reply:
right sorry that's what I meant